### PR TITLE
Add Sanitize examples

### PR DESCRIPTION
**Pull request by kenany**

### CODE REVIEW

**Review by pangea-andrest — DISMISSED** [review]

File: examples/asyncio/sanitize/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+```
+
+Set up the environment variables ([Instructions](https://pangea.cloud/docs/getting-started/integrate/#set-environment-variables)) `PANGEA_SANITIZE_TOKEN` and `PANGEA_DOMAIN` with your project token configured on the Pangea User Console (token should have access to Sanitize service [Instructions](https://pangea.cloud/docs/getting-started/configure-services/#configure-a-pangea-service)) and with your Pangea domain.
+
+
+## Run
+
+To run examples:
+```
+poetry run python sanitize_examples/sanitize_and_share.py
```
[the remark this answers is not in the document]
Comment (kenany) [reply examples/asyncio/sanitize/README.md:20]: So, do you want to see a note about that added to the README or to see this changed to run a different example like `sanitize_examples/sanitize_multipart.py` instead?

[the remark this answers is not in the document]
Comment (pangea-andrest) [reply examples/asyncio/sanitize/README.md:20]: I think I'd prefer to add the note about it on the readme so we are sure that users have all the information to run the examples properly

[the remark this answers is not in the document]
Comment (kenany) [reply examples/asyncio/sanitize/README.md:20]: Added.

**Review by pangea-andrest — APPROVED** [review]
Thanks!

**Pull request merged**